pl_examples/hydra_examples/README.md

```diff
@@ -0,0 +1,8 @@
+## Hydra Pytorch Lightning Example
+
+This directory consists of an example of configuring Pytorch Lightning with (Hydra)[https://hydra.cc/]. Hydra is a tool that allows for the easy configuration of complex applications. 
+
+
+### Tensorboard Visualization
+
+Hydra by default changes the running directory of your program when running into outputs/[YEAR]/[TIME]. Hence, all data with a relative path is submitted into this directory. Therefore to visualize all your tensorboard runs one should run the command: `tensorboard --logdir outputs`. This will then allow you to compare your results across runs. 
```

Comment:
```suggestion
Hydra by default changes the running directory of your program when running into outputs/[YEAR]/[TIME]. Hence, all data with a relative path is submitted into this directory. Therefore to visualize all your tensorboard runs one should run the command: `tensorboard --logdir outputs`. This will then allow you to compare your results across runs.
You can [customize](https://hydra.cc/docs/configure_hydra/workdir) your Hydra working directory.
```